spec/EntitiesSpec.xml

```diff
@@ -5,26 +5,43 @@
 
   -->
   <Entity Name="Entity" SubclassMarker="" TypeString="">
-    <Field Name="handle" Code="5" Type="Handle" DefaultValue="Handle::empty()" ReaderOverride="pair.as_handle()?" />
```

Comment:
This formatting makes it very hard to review. Can you revert the auto formatting of the rest?